app/presenters/job_presenter.rb

```diff
@@ -26,6 +26,16 @@ def list(jobs)
         single(job)
       end.compact
     end
+    
+    def list_by_cluster_id(jobs, cluster_id)
+      jobs.map do |job|
+        next if job['payload'].blank?
+        payload = JSON.parse(job['payload'])
+        if payload['parameters']['integration_id'] == cluster_id
```

Comment:
The parameter name in case of import cluster flow is `TendrlContext.integration_id` so the import cluster job wont be listed with this.